plugins/modules/nmcli.py

```diff
@@ -411,6 +411,14 @@
         type: str
         choices: [ same_all, by_active, only_active ]
         version_added: 3.4.0
+    runner_fast_rate:
+        description:
+            - Option specifies the rate at which our link partner is asked to transmit LACPDU
+              packets. If this is true then packets will be sent once per second. Otherwise they
+              will be sent every 30 seconds.
+            - Only allowed for lacp runner.
```

Comment:
Shouldn't this be capitalized?
```suggestion
            - Only allowed for LACP runner.
```

Reply:
I think ```C(lacp)``` is better